packages/components/src/components/ContentCard/ContentCard.test.tsx

```diff
@@ -0,0 +1,111 @@
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { ContentCard } from "./ContentCard";
+
+describe("<ContentCard>", () => {
+  const user = userEvent.setup();
+
+  it("renders text props", () => {
+    render(
+      <ContentCard
+        date="Wed, Dec 27 2023"
+        href=""
+        imageAlt=""
+        imageSrc=""
+        tag="🏠 housing"
+        text="hello! Long text"
+        title="Good title!"
+      />,
+    );
+
+    expect(screen.queryByText("Good title!")).toBeVisible();
+    expect(screen.queryByText("hello! Long text")).toBeVisible();
+    expect(screen.queryByText("🏠 housing")).toBeVisible();
+    expect(screen.queryByText("Wed, Dec 27 2023")).toBeVisible();
+  });
+
+  it("renders image", () => {
+    render(
+      <ContentCard
+        date=""
+        href=""
+        imageAlt="a test image"
+        imageSrc="path-to-image.jpg"
+        tag=""
+        text=""
+        title=""
+      />,
+    );
```

Comment:
Because the way the types are being [used so far](https://github.com/Localitos/pluto/pull/1209/files#diff-f7fa8c6df160defa8d1aad22a1a3e5af7da525824cca21b138dee5363ceedbd9R20) when there is a href, both linkHref and linkText should be undefined and vice-versa this is not so easily done